### ISSUE TITLE
Generates obfuscated codes with a syntax error when if or else clause contains only one const / let assignment statement

### ISSUE THREAD

**Issue by HealthyPeace** [javascript-obfuscator/javascript-obfuscator#661]
## Example inputs that cause the problem
1. 
```
if (true) {
    const n = 42;
}
```
is obfuscated into
```
if(!![])const n=0x2a;
```

2.
```
if (true) {
} else {
  let n = 42;
}
```
is obfuscated into
```
if(!![]){}else let n=0x2a;
```

When ```if``` or ```else``` clause contains only one statement, ```{``` and ```}``` are omitted in the obfuscated code  and this makes a problematic sequence of words like ```if const <statement>``` or ```else let <statement>```.

## Steps to Reproduce
1. Access https://obfuscator.io/ and leave all the options default.
2. Obfuscate examples above.
3. Input the obfuscated code into Chrome console and see the result.

## Your Environment
Chrome 83.0.4103.116 (Official Build) （64 bit） (cohort: Stable)

**Comment by sanex3339** [comment]
Thank you! As a temporary solution until the fix, disable `simplify` option

**Comment by HealthyPeace** [comment]
Tank you for your very quick response and work around. I can also omit "const n=" part because this only shows the intention of right hand statement and the variable itself is useless because it is located at the end of {} scope.

**Comment by sanex3339** [comment]
Released as `1.5.1`
